src/extensions/scratch3_vizblocks/index.js

```diff
@@ -136,6 +174,346 @@ class Scratch3VizBlocks {
         }
         return penState;
     }
+
+    getInfo () {
+        return {
+            id: 'vizblocks',
+            name: 'VizBlocks',
+            blockIconURI: blockIconURI,
+            blocks: [
+                {
+                    opcode: 'drawAxisX',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'vizblocks.drawAxisX',
+                        default: 'draw axis-X label:[LABEL]',
+                        description: 'draw axis-X and insert label'
+                    }),
+                    arguments: {
+                        LABEL: {
+                            type: ArgumentType.STRING,
+                            defaultValue: 'Insert x-label here'
+                        }
+                    }
+                },
+                {
+                    opcode: 'drawAxisY',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'vizblocks.drawAxisY',
+                        default: 'draw axis-Y label:[LABEL]',
+                        description: 'draw axis-Y and insert label'
+                    }),
+                    arguments: {
+                        LABEL: {
+                            type: ArgumentType.STRING,
+                            defaultValue: 'Insert y-label here'
```

Comment:
Let's try this, the text is out of the screen for long text.

```suggestion
                            defaultValue: 'Insert y-label'
```